### PR TITLE
feature/sf-grouped-product

### PR DESCRIPTION
**Pull request by przspa**
# Related issue

#408 
# Scope of work

[[deploy-preview]](https://deploy-preview-415--storefrontui-storybook.netlify.com/?path=/story/organisms-groupedproduct--default)
After talking with @filrak about SKU props I was created less specific component like SfCollectedProduct.
``` html
<SfGroupedProduct>
  <SfGroupedProductItem
    v-model=""
    :image=""
    :title=""
    :stock=""
    :price-regular=""
    :price-special="">
    <template #configuration>
      <!-- for example SfProperty -->
    </template>
    <template #details>
      <!-- for example SKU like on design-->
    </template>
  </SfGroupedProductItem>
</SfGroupedProduct>
```
*v-model works the sama like in SfCollectedProduct
*We need `SfGroupedProduct` for carousel view on mobile 
# Screenshots of visual changes
[desktop]
Zrzut ekranu 2019-11-8 o 19 09 44
[desktop][with slot's content]
Zrzut ekranu 2019-11-8 o 19 10 02
[mobile]
Zrzut ekranu 2019-11-8 o 19 10 22

# Checklist
- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/organisms/SfGroupedProduct/_internal/SfGroupedProductItem.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<li class="glide__slide sf-grouped-product-item">
+  <div class="sf-grouped-product-item__aside">
+    <slot name="image" v-bind="{image, title}">
+      <SfImage
```
Comment (filrak) [inline packages/vue/src/components/organisms/SfGroupedProduct/_internal/SfGroupedProductItem.html]: we are missing other properties here

**Comment by przspa** [comment]
todo: remove sku

File: packages/shared/styles/components/SfGroupedProduct.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
+@import "../variables";
+
+$grouped-product-max-width: 7.8125rem !default;
+$grouped-product-max-width-desktop: 5.125rem !default;
+$grouped-product-font-family: $body-font-family-secondary !default;
+$grouped-product-font-size: $font-size-regular-mobile !default;
+$grouped-product-font-size-desktop: $font-size-extra-small-desktop !default;
+$grouped-product-font-weight: $body-font-weight-secondary !default;
+$grouped-product-line-height: 1.6 !default;
+$grouped-product__select-quantity-width: 3.125rem !default;
```
Comment (filrak) [inline packages/shared/styles/components/SfGroupedProduct.scss]: do we need those? Shouldn't be this a a general purpose QuantitySelect  component?

File: packages/vue/src/components/organisms/SfGroupedProduct/SfGroupedProduct.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import Vue from "vue";
+import Glide from "@glidejs/glide";
+import SfGroupedProductItem from "./_internal/SfGroupedProductItem.vue";
+
+Vue.component("SfGroupedProductItem", SfGroupedProductItem);
+
+export default {
```
Comment (filrak) [inline packages/vue/src/components/organisms/SfGroupedProduct/SfGroupedProduct.js]: shouldn't we pass glide options as prop like on other components?

File: packages/shared/styles/components/SfGroupedProduct.scss (hunk 2 above)
Comment (przspa) [reply packages/shared/styles/components/SfGroupedProduct.scss]: @filrak definitely we need SfQuantitySelector but now we used SfInput and add styles for it. I add issue for that #453

File: packages/vue/src/components/organisms/SfGroupedProduct/SfGroupedProduct.js (hunk 3 above)
Comment (przspa) [reply packages/vue/src/components/organisms/SfGroupedProduct/SfGroupedProduct.js]: sure :)

File: packages/vue/src/components/organisms/SfGroupedProduct/_internal/SfGroupedProductItem.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+     * Product image
+     */
+    image: {
+      type: String,
+      default: ""
+    },
+    /**
+     * Product image placeholder
+     */
+    placeholderImage: {
```
[the remark this answers is not in the document]
Comment (przspa) [reply packages/vue/src/components/organisms/SfGroupedProduct/_internal/SfGroupedProductItem.js]: refactored, image-x, price-x sounds better

**Comment by przspa** [comment]
@filrak improvemented all changes from your request

**Comment by filrak** [comment]
We are just missing knobs here https://deploy-preview-415--storefrontui-storybook.netlify.com/?path=/story/organisms-groupedproduct--default
Please add them and merge in without my review, everything else is fine ;)

**Pull request merged**